src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageActions.styles.ts

```diff
@@ -106,5 +106,5 @@ export const getActionsMenuCSS = (isActive?: boolean): CSSObject => {
 };
 
 export const messageWithHeaderTop: CSSObject = {
-  top: '45px',
+  top: '-53px',
```

Comment:
This is a little worrying, it there an explanation for this value?

Reply:
this is to position the reaction floating menu with rest to the message header